softwareComponents/kinematics/rofiBot.cpp

```diff
@@ -0,0 +1,185 @@
+#include "rofiBot.hpp"
+
+// executable should probably be separated later
+int main( int argc, char** argv )
```

Comment:
Ideally, for an easy compilation, I would like that the CLI tools end up in `tools` and `softwareComponents/kinematics` is only the library. 

Also, we should choose a suitable CLI parser library and use it all across the project.